frontend/views/Culture.tsx

```diff
@@ -92,7 +92,7 @@ const ExampleInsight = {
  * @returns React Element
  */
 function CultureView(props: Props): React.ReactElement {
-  const { cultureName } = props.route.params;
+  const cultureName = props.route.params ? props.route.params.cultureName : "";
```

Comment:
I'm not sure if this will ever happen, but best to handle the edge case. In this situation this will effectively just navigate to "Home".

Reply:
It was happening to me when I refreshed the page. That's why I added it